python/python/glide/async_commands/core.py

```diff
@@ -2235,6 +2235,39 @@ async def zpopmax(
             ),
         )
 
+    async def bzpopmax(
+        self, keys: List[str], timeout: float
+    ) -> Optional[List[Union[str, float]]]:
+        """
+        Blocks the connection until it removes and returns a member with the highest score from the sorted sets stored
+        at the specified keys. The sorted sets are checked in the order they are provided.
```

Comment:
Please update to be consistent with the Java implementation:
```
     * Blocks the connection until it removes and returns a member with the highest score from the
     * first non-empty sorted set, with the given <code>keys</code> being checked in the order they
     * are provided.
```

Reply:
Arg, I copy-pasted from the original Java PR but it was updated in another one, fixed now